tests/scancode/data/unicodepath/unicodepath.expected-linux.json

```diff
@@ -59,20 +57,19 @@
       "type": "file", 
       "name": "%CF%A8%E1%BD%80%E2%8C%A8a", 
       "extension": "", 
-      "date": "2016-12-05", 
```

Comment:
I just regenerated the test scans hence the new date. In most cases the dates are ignored in tests with this https://github.com/nexB/scancode-toolkit/blob/917b9db7e4fc3bd91811a82e34e50d4d56067f3e/src/scancode/cli_test_utils.py#L51 ... otherwise the tests would never pass